### PR TITLE
remove circular ref

### PR DESCRIPTION
**Pull request by xiaoxuqi-ms**

### CODE REVIEW

File: lib/armValidator/operationLoader.ts [hunk 1]
```diff
@@ -15,6 +15,12 @@ export enum CompareType {
   isMissing,
 }
 
+const statusCodeMap = new Map([
+  ["OK", "200"],
+  ["CREATED", "201"],
+  ["ACCEPTED", "202"],
```
Comment (raych1) [inline lib/armValidator/operationLoader.ts:21]: Should we add 204?

File: lib/swagger/jsonLoader.ts [hunk 2]
```diff
@@ -148,6 +148,9 @@ export class JsonLoader implements Loader<Json> {
             },
           },
         },
+        dereference: {
+          circular: "ignore",
```
Comment (raych1) [inline lib/swagger/jsonLoader.ts:152]: Is this covered in test?

File: lib/armValidator/operationLoader.ts (hunk 1 above)
Comment (xiaoxuqi-ms) [reply lib/armValidator/operationLoader.ts:21]: Good catch!

File: lib/swagger/jsonLoader.ts (hunk 2 above)
Comment (xiaoxuqi-ms) [reply lib/swagger/jsonLoader.ts:152]: Add a test for circular swagger

**Review by raych1 — APPROVED** [review]

**Pull request merged**